cryptography/src/ed25519/scheme.rs

```diff
@@ -302,22 +315,24 @@ impl From<ed25519_consensus::Signature> for Signature {
 }
 
 impl Debug for Signature {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         write!(f, "{}", hex(&self.raw))
     }
 }
 
 impl Display for Signature {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         write!(f, "{}", hex(&self.raw))
     }
 }
 
 /// Ed25519 Batch Verifier.
+#[cfg(feature = "std")]
```

Comment:
unfortunately seems ed25519_consensus::batch requires std.

Reply:
Fair enough, can come back to it if needed.